README.md

```diff
@@ -39,23 +45,34 @@ before_script:
 ```yaml
 matrix:
   include:
-    - rvm: 2.3.1
+    - rvm: 2.3.3
       script:
         - bundle exec danger
 ```
 
-And add the `DANGER_GITHUB_API_TOKEN` as an environment variable in
-Travis's repo settings, which are at a URL like
-`https://travis-ci.org/ministryofjustice/[repo]/settings`. The API token
-itself is in Rattic (search for `Danger`).
+Commit and push the changes.
+
+1. Add the MoJ Dangerbot API token to Travis;
+
+This assumes your project has already been added to Travis.
+
+    https://travis-ci.org/ministryofjustice/[your project]/settings
+
+Add the environment variable `DANGER_GITHUB_API_TOKEN`, with the value
+from the password field of the `MOJ Danger bot Github API token` entry
+in [Rattic](https://rattic.service.dsd.io/cred/detail/695/)
```

Comment:
Rattic's IP-restricted to our building, but I'm not sure we should necessarily be advertising its presence in our readmes? 🤔 Dunno.

@ministryofjustice/secops, any of you have thoughts either way?